published/011-2016-May-16.md

```diff
@@ -82,6 +82,31 @@ Across the entire IPFS GitHub organization, the following people have committed
 
 This newsletter is also a community effort. If you have cool things to share for the next newsletter, drop a comment about it in the [next sprint issue](https://github.com/ipfs/newsletter/issues/43)! The more people mention items they want to see in the newsletter there, the easier it is to make this and send it out.
 
+* [@shazow](https://github.com/shazow) (Andrey Petrov)
+* [@BigBlueHat](https://github.com/BigBlueHat) (BigBlueHat)
+* [@insanity54](https://github.com/insanity54) (Chris Grimmett)
+* [@chriscool](https://github.com/chriscool) (Christian Couder)
+* [@NodeGuy](https://github.com/NodeGuy) (David Braun)
+* [@dominictarr](https://github.com/dominictarr) (Dominic Tarr)
+* [@dylanPowers](https://github.com/dylanPowers) (Dylan Powers)
+* [@AtnNn](https://github.com/AtnNn) (Etienne Laurin)
+* [@harlantwood](https://github.com/harlantwood) (Harlan T Wood)
+* [@cryptix](https://github.com/cryptix) (Henry)
+* [@whyrusleeping](https://github.com/whyrusleeping) (Jeromy)
+* [@jbenet](https://github.com/jbenet) (Juan Batiz-Benet)
+* [@krl](https://github.com/krl) (Kristoffer Ström)
+* [@ehd](https://github.com/ehd) (Stephan Seidt)
+* [@grncdr](https://github.com/grncdr) (Stephen Sugden)
+* [@timgws](https://github.com/timgws) (Tim Groeneveld)
+* [@tv42](https://github.com/tv42) (Tommi Virtanen)
+* [@travisperson](https://github.com/travisperson) (Travis Person)
+* [@vitorbaptista](https://github.com/vitorbaptista) (Vitor Baptista)
+* [@wking](https://github.com/wking) (W. Trevor King)
+* [@bmcorser](https://github.com/bmcorser) (bmcorser)
+* [@krl](https://github.com/krl) (kristoffer)
+
```

Comment:
oh my goodness.  i did a terrible thing.  when I ran the tool, I used 2015, not 2016!  A fix is incoming!